### ISSUE TITLE
Preserve newlines in CSS

### ISSUE THREAD

**Issue by JamesCoyle** [prettydiff/prettydiff#478]
The one thing I hate when beautifying CSS is that I can't add newlines between sections of CSS to split up parts of the file. 

I want to have something like this where a comment can have two newlines before it to add some visual separation:

    selector a {
        property: value;
    }

    selector b {
        property: value;
    }

    /* Group of related selectors */

    selector c {
        property: value;
    }

    selector c.class{
        property: value;
    }

This currently would get beautified to this however:

    selector a {
        property: value;
    }

    selector b {
        property: value;
    }
    /* Group of related selectors */
    selector c  {
        property: value;
    }

    selector c.class {
        property: value;
    }

And it can't seem to decide whether there should be a newline after the comment or not as it seems to toggle with each save. It would be nice if we could make use of whitespace properly for readability.

**Comment by prettydiff** [comment]
What version of Pretty Diff are you using?  When I try the code at http://prettydiff.com/ it seems to work as desired.

**Comment by JamesCoyle** [comment]
Whichever is included with the atom beautify package.

**Comment by prettydiff** [comment]
Yeah, its an old version 1.16.37.  To resolve this problem help this pull request achieve a green build:  https://github.com/Glavin001/atom-beautify/pull/1750

**Comment by JamesCoyle** [comment]
Well that all looks very confusing to me... I'm not familiar with the process so not really sure how I could contribute but I'm glad it's being worked on.

**Comment by prettydiff** [comment]
When I was working on that about 2 months ago the problem seemed to be conflicts from unrelated packages that were impacting the code test samples in ways that were underwhelming.  There are numerous bugs that have been resolved between the older version and the current version and I would love to get the newer version in there, but I kept running into walls.

At any rate I will start working on Pretty Diff 3 soon.  It will use https://github.com/Unibeautify/parse-framework as its parser and will be much faster to update and maintain with smaller code that does more.
